modules/users/tests/client/password-verify.client.directive.tests.js

```diff
@@ -26,7 +26,7 @@
     function compileDirective(template) {
       // function to compile a fresh directive with the given template, or a default one
       // input form with directive
-      if (!template) template = '<input type="password" id="newPassword" name="newPassword" class="form-control" ng-model="passwordMock.newPassword" placeholder="New Password" autocomplete="new-password" uib-popover="{{popoverMsg}}" uib-popover-trigger="focus" uib-popover-placement="top" password-validator required>' +
+      if (!template) template = '<input type="password" id="newPassword" name="newPassword" class="form-control" ng-model="passwordMock.newPassword" placeholder="New Password" autocomplete="new-password" uib-popover="{{getPopoverMsg}}" uib-popover-trigger="focus" uib-popover-placement="top" password-validator required>' +
```

Comment:
I don't think `vm` is relevant here. We're only testing the directive, and there's no context for a *controller* here. IIRC, from working with these tests, I think this is the correct way to write isolated tests for directives.